src/components/ViolenceTestCards/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Col, Row } from "react-bootstrap";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const ViolenceTestCards = () => {
+  const { t } = useTranslation();
+  const cardsArr = [...t("violenceTestPage.testSpec", { returnObjects: true })];
+  const checkboxItemsArr = [
+    [
+      ...t("violenceTestPage.testSpec.0.checkboxItems", {
+        returnObjects: true,
+      }),
+    ],
+    [
+      ...t("violenceTestPage.testSpec.1.checkboxItems", {
+        returnObjects: true,
+      }),
+    ],
+    [
+      ...t("violenceTestPage.testSpec.2.checkboxItems", {
+        returnObjects: true,
+      }),
+    ],
+  ];
+
+  const cards = cardsArr.map((card) => {
+    return (
+      <div className="col" key={card.id}>
+        <h3>{t(card.category)}</h3>
+        <h5>{t(card.subtitle)}</h5>
+      </div>
+    );
+  });
+
+  const checkboxItems = checkboxItemsArr.map((items) => {
+    return (
+      <form className="test col" key={items.id}>
```

Comment:
is there an id field on these? i dont think it has one

Reply:
in the json at least

Reply:
I thought it fix this warning Each child in a list should have a unique "key" prop.

Reply:
I removed it